src/pages/index.js

```diff
@@ -24,10 +30,27 @@ const profileEditForm = profileEditModal.querySelector(".modal__form");
 const addNewCardButton = document.querySelector(".profile__add-button");
 const addCardModal = document.querySelector("#add-card-modal");
 const addCardForm = addCardModal.querySelector(".modal__form");
+const deleteCardModal = document.querySelector("#delete-card-modal");
 const imageModal = document.querySelector("#image-preview");
+const avatarEditModal = document.querySelector("#avatar-edit-modal");
+const avatarEditButton = document.querySelector("#profile-image-button");
+const avatarEditForm = avatarEditModal.querySelector(".modal__form");
+const avatar = document.querySelector(".profile__image");
 
-addNewCardButton.addEventListener("click", () => {
-  addCardPopup.open();
+const userInfo = new UserInfo(profileTitle, profileDescription, avatar);
+let cardSection;
+
+api.getAppInfo().then(([cards, user]) => {
+  cardSection = new Section(
+    {
+      renderer: renderCard,
+      items: cards,
+    },
+    selectors.cardSection
+  );
+  cardSection.renderItems();
+  userInfo.setUserInfo(user.name, user.about);
+  userInfo.setAvatar(user.avatar);
 });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**